nodejs/README.md

```diff
@@ -33,42 +33,72 @@ Execute on the command line:
 sudo apt install n
 ```
 
-## 2 Install Node.js through `n`
+## 2 Configure
+
+To avoid requiring `sudo` for `n` and `npm` global installs, configure
+environment variable `N_PREFIX`, which will point to the place where `n` will
+store Node.js binaries. With this you will also need to update the `PATH`
+environment variable, so that the system can find the active binaries. Execute
+on the command line:
+
+```console
+cd ~
+mkdir n
+```
+
+You can now update environment variables. In case of `zsh`, add to the
+`~/.zprofile` file:
+
+```shell
+export N_PREFIX=~/"n"
+export PATH=~/"n/bin:$PATH"
+```
+
+Make sure to preserve any existing modifications of the `PATH` variable.
```

Comment:
@pspanja This sentence got me a bit confused, I wasn't sure is there something else I need to take care of if I already have `PATH` modified. I presume you wanted to emphasize that we are adding to the end of the `.zprofile` and not overwriting existing modifications. If so, I would rather modify the line before
_In case of `zsh`, **add to the end of the** `~/.zprofile` file:_
and remove the _Make sure..._ sentence.

Reply:
@pspanja Also, maybe add a reminder that we need to restart terminal to be able to successfully execute the next step?

Reply:
Agreed, updated in https://github.com/netgen/local-development-setup/pull/14/commits/a0ee822dd99305a37a4992e7e12bfdb860cab236